cabal-testsuite/PackageTests/HaddockProject/haddock-project.out

```diff
@@ -2,6 +2,8 @@
 Downloading the latest package list from test-local-repo
 # cabal haddock-project
 Warning: haddock-project command is experimental, it might break in the future
+Configuration is affected by the following files:
+- cabal.project
```

Comment:
Why is this appearing twice in the output?

Reply:
@philderbeast: any comment?

Reply:
Sorry I didn't have time to look at the earlier. Looking now. Please also see https://github.com/haskell/cabal/pull/10940#discussion_r2150431070.